docker/php.Dockerfile

```diff
@@ -46,16 +46,10 @@ RUN docker-php-source extract && \
     mkdir /usr/src/php/ext/amqp && \
     tar --extract --file amqp.tar.gz --directory /usr/src/php/ext/amqp --strip 1 && \
     docker-php-ext-install -j$(nproc) amqp && \
-    docker-php-source delete
-
-RUN docker-php-source extract && \
     wget -O redis.tar.gz https://github.com/phpredis/phpredis/archive/refs/tags/5.3.7.tar.gz && \
     mkdir /usr/src/php/ext/redis && \
     tar --extract --file redis.tar.gz --directory /usr/src/php/ext/redis --strip 1 && \
     docker-php-ext-install -j$(nproc) redis && \
-    docker-php-source delete
-
-RUN docker-php-source extract && \
```

Comment:
What is the reason for removing these commands? (2)

Reply:
If you look at the `docker-php-source` script, it will check if the source already exists before extracting it again, so there was no need to do a sequence of extract/delete/extract :-)